libr/bin/dwarf.c

```diff
@@ -85,7 +85,6 @@ static const char *dwarf_tag_name_encodings[] = {
 	[DW_TAG_subprogram] = "DW_TAG_subprogram",
 	[DW_TAG_template_type_param] = "DW_TAG_template_type_param",
 	[DW_TAG_template_value_param] = "DW_TAG_template_value_param",
-	[DW_TAG_template_alias] = "DW_TAG_template_alias",
```

Comment:
i think it will be good to reorder all those entries by sorting them alphabetically `%!sort` in vim

Reply:
The order of the macros corresponds to the order they appear in r_bin_dwarf.h (and the values assigned). If we sort in dwarf.c then what about the header?